src/Utils/Utils.php

```diff
@@ -149,15 +150,23 @@ public static function getCleanShopwareVersion(): string
      */
     protected static function getShopwareVersion(): string
     {
-        $packageVersions = Versions::VERSIONS;
-        $coreIsInstalled = isset($packageVersions['shopware/core']);
-
-        $shopwareVersion = $coreIsInstalled ?
-            Versions::getVersion('shopware/core') :
-            Versions::getVersion('shopware/platform');
-
-        if (!trim($shopwareVersion, 'v@')) {
-            return $coreIsInstalled ? $packageVersions['shopware/core'] : $packageVersions['shopware/platform'];
```

Comment:
In the past there had been issues with using `Versions::getVersion('shopware/platform')`, therefore getting them directly from the `$packageVersions` is the safer option. See [this commit](https://github.com/findologic/plugin-shopware-6/commit/c463da773960c8125b8d06a6451e09032f127a18) also, in which this has been explicitly opted against.